### PR TITLE
Solved syntax error because of formatting.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by tuliplab** [comment]
Please leave detailed change reason and justify the need for changes.

Also you need to clear all Python output and send the Pull Request for the cleaned notebook

**Comment by ghost** [comment]
There was a syntax error in the `if...else` block, moreover, I used nbstripout to only show necessary diff in the git.

**Comment by ghost** [comment]
Hi there,
Many thanks for your feedback and contribution for the lab materials on weekend.

For the Jupyter notebook format, you can use the ” nbdiff-web tool “ to compare two .ipynb format files, it will the differences directly based on the webpage form, not the git. If you would to learn it, please refer it on the webpage https://github.com/jupyter/nbdime. Please note that how to use “nbdiff-web tool” is not a compulsory point we need to teach for this course.


In addition, for your question “was a syntax error in the if...else block”, can you point out where is the syntax?

If you mean the 1st sample (please check the below codes), you should get an indentation error message from Python interpreter but it is not a bug in the lab materials.

If not, please provide the details to help us to confirm it.

Due to we would like to provide a case unfollowing the Python syntax so that those students who have Zero python programming background can understand the Python syntax deeply especially for what is the exact meaning for the Python indentation sensitive feature? If you have done the first week’s materials(P01A, B, C, D), you will find that we usually provide some example involved errors. Meantime, we also explain the cause for this error above the related codes. For example, in the if…else sample, you can see that:

1.1 If statements
The if statement is used to check a condition: if the condition is true, we run a block of statements(if-block), else we process another block of the statement(else-block). The else clause is optional. The condition is usually a boolean expression, which can have a value of True or False.
Often we have a block of statement inside either if-block or else-block. In this case, you need to especially pay attention to the indentation of the statements in the block. Indenting starts a block and unindenting ends it. As a general practice, the Python style guide recommends using 4 white spaces for indenting a block, and not using tabs. If you get an indentation error message from Python interpreter, you will need to check your code carefully.
x = 15
if x % 2 == 0:
   print('%d is even' % x)
                else:
print('%d is odd' % x)
print('This is always printed')


To sum up, firstly, we really appreciate your contribution to the lab materials quality. secondly, please read the lab materials carefully and then run the code. If not like this, just click the ‘Run button’ for the python code, you will miss many interesting tips on it.

Thanks a lot.



Important Notice: The contents of this email are intended solely for the named addressee and are confidential; any unauthorised use, reproduction or storage of the contents is expressly prohibited. If you have received this email in error, please delete it and any attachments immediately and advise the sender by return email or telephone.

Deakin University does not warrant that this email and any attachments are error or virus free.

**Comment by ghost** [comment]
Thank you for your reply. I am installing `nbdime` and then make commits if required.
